src/SingleLineTextEdit.cpp

```diff
@@ -86,3 +91,7 @@ void SingleLineTextEdit::rehighlight()
 
     highlighter->rehighlight();
 }
+
+bool SingleLineTextEdit::textEdited(){
```

Comment:
Please start methods (functions) with the opening `{`on the next line to match our coding style:
```suggestion
bool SingleLineTextEdit::textEdited()
{
```